--- /var/www/pydio/conf/bootstrap_plugins.bridges.php	2016-10-18 15:12:52.211345326 -0300
+++ /var/www/pydio/conf/bootstrap_plugins.bridges.php.updated.php	2016-10-18 15:14:37.785661650 -0300
@@ -61,6 +61,7 @@
                    )
                )
        ),
+/*
     "AUTH_DRIVER" => array(
            "NAME"		=> "remote",
            "OPTIONS"	=> array(
@@ -75,7 +76,8 @@
                "TRANSMIT_CLEAR_PASS"   => true // Don't touch this. It's unsafe (and useless here) to transmit clear password.
            )
        ),
-    /*
+*/
+
        // Same for Drupal 7.X
        "AUTH_DRIVER"  => array(
            "NAME" => "remote",
@@ -92,7 +94,7 @@
                "TRANSMIT_CLEAR_PASS" => true
                )
        ),
-    */
+
     "LOG_DRIVER" => array(
          "NAME" => "text",
          "OPTIONS" => array(
